config/cloud.cfg.tmpl

```diff
@@ -32,9 +33,9 @@ disable_root: false
 disable_root: true
 {% endif %}
 
-{% if variant in ["almalinux", "alpine", "amazon", "centos", "cloudlinux", "eurolinux",
-                  "fedora", "miraclelinux", "openEuler", "openmandriva", "rhel", "rocky", "virtuozzo"] %}
-{% if variant == "rhel" %}
+{% if variant in ["almalinux", "alpine", "amazon", "cloudlinux", "eurolinux",
+                  "fedora", "miraclelinux", "openEuler", "openmandriva", "rocky", "virtuozzo"] or is_rhel %}
```

Comment:
`almalinux`, `cloudlinux`, `eurolinux`, `miraclelinux`, `rocky`, and `virtuozzo` are all `is_rhel` variants, so they need to be pulled out too.

Reply:
This is question of what we want to have. Originally 'rhel' variant was used exclusively and separated from other variants Now we are making 'centos' equal to it as for CentOS 9 Stream/RHEL 9 it works that way (C9S is synced to RHEL9 as 1:1 copy). Whether this behavior should be applied to other variants is questionable - I do not know what's the general expectation here.

My opinion is: for template treat 'rhel' and 'centos' as equivalent (as they should be from development point of view). Any other variant should use --variant option to say they try to be 'rhel'-like.

However, for RHEL/C9S spec we should probably add this option too to prevent any differences in case centos-rhel equality is not handled properly somewhere in template.